spec/models/email_alert_signup_spec.rb

```diff
@@ -0,0 +1,80 @@
+require "spec_helper"
+
+describe EmailAlertSignup do
+
+  let (:content_item) {
+    OpenStruct.new(
```

Comment:
Would it be possible to write this spec to use the example from `govuk-content-schemas`? That way, it will always run against the latest version of an email-alert-signup, giving us coverage against breaking changes to the format.